docs/splines.md

```diff
@@ -19,49 +18,63 @@ Arrow properties that can be spline'd:
 * Stealth
 * Skew
 
-# How to Apply:
-Call it on Playfield.
+# How to Apply
+Call it on a playfield as follows:
 
 ```lua
 P1:SetXSpline(
 	which,
-	column(-1 for all columns),
+	column, --(-1 for all columns)
 	mod percentage,
-	offset (distance from recep in beats)
-	activation speed (set -1 for instant)
+	offset, --(distance from receptors in beats)
+	activation speed --(set -1 for instant; measured in rates)
 )
 ```
+For activation speed, it's measured in rates, which is just 1 divided by the amount of time you want it to take in seconds. So 1 second would be 100, 2 seconds would be 50, and half a second would be 200.
```

Comment:
I don't think "Measured in rates" is the right way to word this. In all docs about activation rate, I haven't seen the term 'rate' being used as a unit ever. I'm not sure if we want an explanation about rates at all. If we do include any info about rates, I think it would also be good to more clearly indicate that using activation rates is discouraged, and that -1 is recommended.

Reply:
So should we just say the conversion formula and not elaborate on why? Or just not mention it at all?